levels/levels.go

```diff
@@ -56,60 +60,73 @@ func NewChallenge(name string) Challenge {
 	c := Challenge{
 		Name:         name,
 		conf:         cfg,
-		CurrentLevel: flag.String("level", IndexToID(0, name), "Current Level"),
+		CurrentLevel: flag.String("level", LevelToID("", name), "Current Level"),
+		Printed:      flag.String("printed", "no", "Printed"),
```

Comment:
@mrshu I do not seem to get the motivation behind this. Is this here just so the last message does not get printed twice? If so would it be so bad if the last message got printed again? I think it would be rather nice. But if you disagree or I missed something or not fully understand it I'm sorry and I'm fine with it but why is it a string rather than boolean? And could it get some better name then `Printed`?  It does not say much about the function of it maybe something like `LastMessage` or `DisplayLastMessage`.